src/Microsoft.ApplicationInsights.AspNetCore/Extensions/ApplicationInsightsExtensions.cs

```diff
@@ -134,10 +134,10 @@ public static IServiceCollection AddApplicationInsightsTelemetry(this IServiceCo
             services.AddSingleton<ITelemetryInitializer, WebSessionTelemetryInitializer>();
             services.AddSingleton<ITelemetryInitializer, WebUserTelemetryInitializer>();
             services.AddSingleton<ITelemetryInitializer, AspNetCoreEnvironmentTelemetryInitializer>();
+            services.AddSingleton<ITelemetryModule, DependencyTrackingTelemetryModule>();
```

Comment:
please also add `HttpDependenciesParsingTelemetryInitializer ` (see #401 and https://github.com/Microsoft/ApplicationInsights-dotnet-server/issues/482 for motivation)